index.html

```diff
@@ -89,6 +96,9 @@ <h4>Instructions:</h4>
             <option value="surface">surface</option>
           </select>
           <button id="protein_plot_download" type="button" class="btn btn-light float-right"><img src="/assets/images/download.svg" /> Download PNG</button>
+          <label><input type="checkbox" id="colorCheckbox" name="colorCheckbox" value="colorChains"></input>color options</label>
+          <p><input id="protein-data-color" name='protein-data-color' text="text" class="form-control" placeholder="color for data chains" style="display: none"/></p>
+          <p><input id="protein-other-color" name='protein-other-color' text="text" class="form-control" placeholder="color for other chains" style="display: none"/></p>
```

Comment:
What if the “color options” toggle was styled more like a dropdown menu so users would be more likely to expect that they could click the color options to get a menu of settings? For example, we could use [the bootstrap dropdown class with a form inside to get the same effect](https://getbootstrap.com/docs/4.5/components/dropdowns/#forms).